src/main/java/com/ericsson/ei/frontend/BackEndInformationController.java

```diff
@@ -59,15 +59,15 @@
     @CrossOrigin
     @RequestMapping(value = "/backend", method = RequestMethod.DELETE)
     public ResponseEntity<String> deleteBackEndInstance(Model model, HttpServletRequest request) {
-        LOG.debug("Recieved request to delete back end.");
+        LOG.debug("Recieved request to delete EI back end instance.");
```

Comment:
Recieved --> Received . Double check in other log messages also.

Reply:
done.